Mathlib/MeasureTheory/Function/ConditionalExpectation/Real.lean

```diff
@@ -200,7 +200,7 @@ theorem Integrable.uniformIntegrable_condExp {ι : Type*} [IsFiniteMeasure μ] {
   have hCpos : 0 < C := mul_pos (inv_pos.2 hδ) (ENNReal.toNNReal_pos hne hg.eLpNorm_lt_top.ne)
   have : ∀ n, μ {x : α | C ≤ ‖(μ[g|ℱ n]) x‖₊} ≤ ENNReal.ofReal δ := by
     intro n
-    have := mul_meas_ge_le_pow_eLpNorm' μ one_ne_zero ENNReal.one_ne_top
+    have := mul_meas_ge_le_pow_eLpNorm'' μ one_ne_zero ENNReal.one_ne_top
```

Comment:
If you replace this with the new version, can't you just (simp-)rewrite with `enorm_eq_nnnorm` to get exactly the old version?

Reply:
I couldn't "just" do it - but trying harder, this indeed worked. I removed the doubly-primed lemma now.

Reply:
Looking at the diff, the effect is not exactly pretty. I'll this PR as-is, but if there is a preference to revert, am happy to re-introduce the old version instead. (Locally rewriting this file is not an easy option; I checked. One would have to go over "everything", which should definitely be a follow-up.)